mne/viz/utils.py

```diff
@@ -1032,3 +1036,102 @@ def _plot_sensors(pos, colors, ch_names, title, show_names, show):
     fig.suptitle(title)
     plt_show(show)
     return fig
+
+
+def compute_scalings(scalings, inst):
+    """Compute scalings for each channel type automatically.
+
+    Parameters
+    ----------
+    scalings : dict
+        The scalings for each channel type. If any values are
+        'auto', this will automatically compute a reasonable
+        scaling for that channel type.
+    inst : instance of Raw or Epochs
+        The data you wish to compute scalings for. If Epochs,
```

Comment:
I just really enjoy exposing my knowledge of English grammar to the world. It's in response to the frustration Reviewers #2 generate, when asking for improving the language of my manuscripts.